integreat_compass/cms/templates/offers/offer_report.html

```diff
@@ -0,0 +1,32 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% load render_bundle from webpack_loader %}
+{% block title %}
+    {% blocktranslate trimmed with offer_title=selected_offer.public_version.title %}
+        Report offer "{{ offer_title }}"
+    {% endblocktranslate %}
+{% endblock title %}
+{% block content %}
+    <div class="w-full flex flex-wrap justify-between mb-6 gap-2">
+        <h1 class="heading">
+            {% blocktranslate trimmed with offer_title=selected_offer.public_version.title %}
+                Report offer "{{ offer_title }}"
+            {% endblocktranslate %}
+        </h1>
+    </div>
+    <div class="flex items-center">
+        <i class="text-primary" icon-name="chevron-left"></i>
+        <a class="text-primary hover:cursor-pointer font-bold hover:underline"
+           href="{% url 'cms:public:index' %}">{% translate "Go back to all offers" %}</a>
```

Comment:
IMO this link has to redirect to the filtered view the user was seeing previously. Currently all filters are lost.

The proper way would probably be to pass the previous URL along when hitting the report button. An easier (albeit a bit "dirtier") way could be to just use JS and do `history.back()`, though if you find the time, I'd prefer the former method 😅

Reply:
Good idea! Only issue I am seeing right now is that after successfully submitting the report, that link now points to the reporting URL.

Reply:
Ah damn, yes. I will have a look at it

Reply:
This is now also resolved as discussed in the weekly meeting. If the submitting is successful the user gets redirected to the index page and gets the success message there. The issue for the overlapping filter bar is already created, see [here](https://github.com/digitalfabrik/integreat-compass/issues/100).